docker-compose.dev.yml

```diff
@@ -1,16 +1,43 @@
 services:
-  virtuoso:
+  identifier:
+    restart: "no"
+  dispatcher:
+    restart: "no"
+    ports:
+      - "80:80"
```

Comment:
Is the exposing of the stack for development through `dispatcher` on purpose? I'd think that should be identifier. Since the identifier takes care of some of the session stuff (passing on headers deeper down the stack), this will result in sessions not behaving as expected sooner or later, no?